zoo.py

```diff
@@ -0,0 +1,30 @@
+
+class Animal:
+    def __init__(self, name):
+        self.name = name
+        self.age = 2
+
+    def speak(self):
+        print(self.age)
```

Comment:
You should print("My age is ", self.age)

Reply:
made the change